python/py-sphinxnotes-strike/Portfile

```diff
@@ -17,15 +17,20 @@ long_description    {*}${description}
 
 homepage            https://sphinx-notes.github.io/strike
 
-checksums           sha256  68f3e1cd152476421f17d85568aeea67f3c0eac7271ae33d27f1194c3841e2b0 \
-                    rmd160  206733513d3b92efa42340ad49bd9dff6916a368 \
-                    size    3723
+checksums           sha256  d51b5b8b60c62ba647abe9d5f32e78a9a4feddb48bf76aa77bbc4b6d5d968e70 \
+                    rmd160  b511db430f75354e5657520d23dd3d5bdbf2c4bb \
+                    size    29117
 
-python.versions     37 38 39 310
+python.versions     38 39 310 311 312
 
 if {${name} ne ${subport}} {
-    depends_build-append \
+    if {${python.pep517}} {
+        depends_build-append \
+                    port:py${python.version}-setuptools_scm
+    } else {
+        depends_build-append \
                     port:py${python.version}-setuptools
+    }
```

Comment:
`python.pep517` is always `yes` for Python 3.7+ subports, do this `if-else` block is unnecessary

Reply:
Thanks, fixed.